WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/ReaderPage.kt

```diff
@@ -16,6 +16,13 @@ class ReaderPage {
         return this
     }
 
+    fun openBlog(blogName: String?): ReaderViewPage {
```

Comment:
looks like openBlog and openPost both do exactly the same thing. maybe we could rename them something more generic so that it's clear it can be used for both cases?

```suggestion
    fun openBlogOrPost(name: String?): ReaderViewPage {
```

Reply:
Actually, they use different `scrollableContainerID`s because one of them was running on the Following tab, and the other one was on the Discover tab. It was confusing. So, I changed it and used the Following tab for also bookmark test. 8c09a3678a4c5ebaede2a90c9fd7ed5dd3e77e79